test-utils/test-crate/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "substrate-test-utils-test-crate"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "Apache-2.0"
+homepage = "https://substrate.dev"
+repository = "https://github.com/paritytech/substrate/"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dev-dependencies]
+tokio = { version = "0.2.13", features = ["macros"] }
```

Comment:
@bkchr I made this test-crate to check if it worked but I found out that tokio relies on the extern name tokio. The rest is fine and working though. I inspired my code from what you provided but I simplified a bit.

(I have not done the generate_crate_access yet!)

Reply:
I'm not sure what generate_crate_access is doing/solving. Do you have a small & quick explanation?